Content.Server/GameTicking/Rules/ThiefRuleSystem.cs

```diff
@@ -101,6 +102,27 @@ public void MakeThief(EntityUid thief, ThiefRuleComponent thiefRule, bool addPac
 
         thiefRule.ThievesMinds.Add(mindId);
     }
+    
+    private void HandleLatejoin(PlayerSpawnCompleteEvent ev)
+    {
+      var query = QueryActiveRules();
+      while (query.MoveNext(out _, out var comp, out _))
+      {
+        if (comp.ThievesMinds.Count >= comp.MaxAllowThief)
+          continue;
+
+        if (!ev.LateJoin)
+          continue;
+        
+        if(!_antagSelection.IsPlayerEligible(ev.Player, comp.ThiefPrototypeId))
+          continue;
+        
+        if (_random.Prob(0.90f))
+          continue;
```

Comment:
When the maximum amount of thieves isn't achieved yet, the chance of randomly rolling thieves on late join is 10% based on this code. 

Should it be lowered? and if so? to what?